path/resolver.go

```diff
@@ -135,6 +135,7 @@ func (s *Resolver) ResolvePathComponents(ctx context.Context, fpath Path) ([]nod
 	if err != nil {
 		return nil, err
 	}
+	defer log.EventBegin(ctx, "resolvePathComponents", logging.LoggableMap{"parts": parts, "cid": h}).Done()
```

Comment:
In the case that there is an error in the execution of this method, do we want to add a loggable error to the event log (see L143)?

Reply:
IMO, yes. We'd probably also want to log the error from L146 as well. Also, this event should probably go at the top and include any errors from `SplitAbsPath`.

Reply:
You can save the result value of `EventBegin` and add additional data to it.